pkg/providers/snow/reconciler/reconciler.go

```diff
@@ -75,11 +104,10 @@ func (s *Reconciler) ReconcileWorkers(ctx context.Context, log logr.Logger, clus
 	})
 }
 
-func (s *Reconciler) ReconcileControlPlane(ctx context.Context, log logr.Logger, clusterSpec *cluster.Spec) (controller.Result, error) {
-	log = log.WithValues("phase", "reconcileControlPlane")
-	log.Info("Applying control plane CAPI objects")
-
-	return s.Apply(ctx, func() ([]kubernetes.Object, error) {
-		return snow.ControlPlaneObjects(ctx, clusterSpec, clientutil.NewKubeClient(s.client))
-	})
+func capiClusterObjectKey(cluster *anywherev1.Cluster) client.ObjectKey {
```

Comment:
Should we also define this in one of the capi files that you are introducing in #3193? 
Maybe in `pkg/controller/clusters/clusterapi.go`

Reply:
we probably should yeah
what about we make this a follow up? I don't have time right now to move it and I want to start the testing as soon as possible. Or maybe you can move the method and add a test? And just push to this branch :)

Reply:
Talked about it offline, we will have a follow up PR